From 9be51b563c98744053e4a7412e5030fa2ab3e061 Mon Sep 17 00:00:00 2001
From: Marius Vollmer <mvollmer@redhat.com>
Date: Mon, 8 Aug 2022 14:50:51 +0300
Subject: [PATCH] kdump: Use close with cockpit.file, not remove

There is no remove, and the intention is to close the watch channel.
---
 pkg/kdump/config-client.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/kdump/config-client.js b/pkg/kdump/config-client.js
index 9b95b9a0c65..c58cb85fd99 100644
--- a/pkg/kdump/config-client.js
+++ b/pkg/kdump/config-client.js
@@ -47,7 +47,7 @@ export class ConfigFile {
 
     close() {
         if (this._fileHandle) {
-            this._fileHandle.remove();
+            this._fileHandle.close();
             this._fileHandle = undefined;
         }
     }
